static/scripts/app-login.js

```diff
@@ -33,46 +38,41 @@ formbtn.addEventListener('click', async (e) => {
           },
         body: jsonData
     });
-    switch (datafetch.status) {
-        case 200:
-            console.log('successful login');
-            location.reload();
-            return;
-        case 302:
-        case 303:
-            console.log('repetitive login attempt was redirected');
-            location.reload();
-            return;
-        case 400:
-            console.log(datafetch.statusText, 'Empty values.')
-            return
-        case 401:
-            const inputsContainer = document.querySelector('.inputs-container');
-            let user_err = document.createElement('small');
-                    user_err.classList.add('big-user-err');
-                    user_err.innerText = `Invalid username or password.`;
-                    // remove err if already exist
-                    if(inputsContainer.firstChild.classList?.contains('big-user-err')){
-                        inputsContainer.firstChild.remove();
-                    }
-                    inputsContainer.prepend(user_err);
-
-                    setTimeout(() => {
-                        user_err.remove();
-                    }, 5000);
-            console.log(datafetch.statusText, 'Invalid username or password.')
-            return
-        case 500:
-            console.log(datafetch.statusText, '')
-            return
-        default:
-            console.log(`unexpected status received: ${datafetch.status}`)
-    }
-    let result = await datafetch.json();
-    console.log(result)
-    console.log(jsonData)
 
+    const result = await datafetch.json();
     
+    if(datafetch.ok){
```

Comment:
Good decision to reduce the # of paths in the control. My switch approach was redundant as we only care about OK vs Not OK and then in case of the latter rendering msg to the clientele.

Reply:
agree